datastore/protocol.go

```diff
@@ -47,8 +47,8 @@ const (
 	// MethodSearchObject is the method ID for the method SearchObject
 	MethodSearchObject = 0xC
 
-	// MethodGetNotificationURL is the method ID for the method GetNotificationUrl
-	MethodGetNotificationURL = 0xD
+	// MethodGetNotificationUrl is the method ID for the method GetNotificationUrl
+	MethodGetNotificationUrl = 0xD
```

Comment:
Why was this changed? Go's style guide dictates that initialisms/acronyms have consistent casing. So for example either `url` or `URL`, but not `Url`. Since functions use camel casing, that would mean `Url` becomes `URL`